commands/droplets.go

```diff
@@ -43,13 +43,13 @@ func Droplet() *Command {
 		IsIndex:       true,
 	}
 
-	CmdBuilder(cmd, RunDropletActions, "actions <droplet id>", "droplet actions", Writer,
+	CmdBuilder(cmd, RunDropletActions, "actions <droplet-id>", "droplet actions", Writer,
 		aliasOpt("a"), displayerType(&action{}), docCategories("droplet"))
 
-	CmdBuilder(cmd, RunDropletBackups, "backups <droplet id>", "droplet backups", Writer,
+	CmdBuilder(cmd, RunDropletBackups, "backups <droplet-id>", "droplet backups", Writer,
 		aliasOpt("b"), displayerType(&image{}), docCategories("droplet"))
 
-	cmdDropletCreate := CmdBuilder(cmd, RunDropletCreate, "create NAME [NAME ...]", "create droplet", Writer,
+	cmdDropletCreate := CmdBuilder(cmd, RunDropletCreate, "create <droplet-name>", "create droplet", Writer,
```

Comment:
Perhaps `create <droplet-name> [<droplet-name> ...]` to denote 1 or more arguments?